parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -313,6 +378,16 @@ cleanUpContainerImages() {
     bash -c cleanUpKubeProxyImages &
 }
 
+removeContainerImage() {
```

Comment:
I don't see where we use this?

Reply:
being used now in cleanupKubeProxy and cleanupHyperkube

Reply:
also used in install-dependencies